charmcraft.yaml

```diff
@@ -5,10 +5,15 @@ type: charm
 bases:
   - build-on:
     - name: "ubuntu"
-      channel: "20.04"
+      channel: "22.04"
+      architectures: ["amd64"]
     run-on:
+    - name: "ubuntu"
+      channel: "22.04"
+      architectures: ["amd64"]
     - name: "ubuntu"
       channel: "20.04"
+      architectures: ["amd64"]
```

Comment:
Not sure if this 20.04 section is needed.

Reply:
It is.